release.yml

```diff
@@ -127,7 +127,10 @@ stages:
         artifactName: FhirConverterBuild
         downloadPath: $(System.DefaultWorkingDirectory)
     - script: |
-        docker run --rm -d -p 5000:5000 --name registry stefanscherer/registry-windows:2.6.2
+        git clone -q https://github.com/sowu880/dockerfiles-windows.git
+        cd dockerfiles-windows/registry/
+        docker build -t registry-windows:2.7.1 .
+        docker run --rm -d -p 5000:5000 --name registry registry-windows:2.7.1
```

Comment:
Just for confirmation, if we run a docker in daemon mode, will the azure pipeline wait for the container to finish running?

Reply:
By teams discussion there is no issue for the docker-in-pipeline :)